src/CLI/klish/patches/klish-2.1.4/clish/plugin/module.am.diff

```diff
@@ -0,0 +1,6 @@
+5c5,7
+< 	clish/plugin/private.h
+---
+> 	clish/plugin/private.h \
+> 	clish/plugin/clsh_api.h
```

Comment:
This file looks to be spelled incorrectly; should be `clish_api.h` not `clsh_api.h`.
This should cause a compilation failure, no?

Reply:
Thanks for the catch. There is no need to add the header file in module.am since it is implicitly imported to the C code. The file is removed.

Reply:
The mis-spelled header file is ignored. The correct file is directly imported to C code. This file is now removed to honor the the module.am as is from Klish open source.